### ISSUE TITLE
reproduce resnet results on CIFAR10

### ISSUE THREAD

**Issue by shamafiras** [christiancosgrove/pytorch-spectral-normalization-gan#10]
Hey,
I'm trying to reproduce the published results of the resnet network on the CIFAR10 dataset.
As published in the paper, they were able to achieve a score of 8.22 using resnet network, however I couldn't achieve above 7.5+- in this net. Am I missing something ?
I tried the 3 available loss options, only the models trained on bce were converged, while the other collapsed.
Thanks

**Comment by WhiteTeaDragon** [comment]
Is there any news? Was anyone able to reproduce ResNet + SN results?
